examples/guided.py

```diff
@@ -263,42 +349,55 @@ def perform_installation_steps():
 		with archinstall.Filesystem(archinstall.arguments['harddrive'], mode) as fs:
 			# Wipe the entire drive if the disk flag `keep_partitions`is False.
 			if archinstall.arguments['harddrive'].keep_partitions is False:
-				fs.use_entire_disk(root_filesystem_type=archinstall.arguments.get('filesystem', 'btrfs'))
-			
+				fs.use_entire_disk(
+					root_filesystem_type=archinstall.arguments.get('filesystem', 'btrfs')
+				)
+
 			# Check if encryption is desired and mark the root partition as encrypted.
 			if archinstall.arguments.get('!encryption-password', None):
 				root_partition = fs.find_partition('/')
 				root_partition.encrypted = True
-					
+
 			# After the disk is ready, iterate the partitions and check
 			# which ones are safe to format, and format those.
 			for partition in archinstall.arguments['harddrive']:
 				if partition.safe_to_format():
 					# Partition might be marked as encrypted due to the filesystem type crypt_LUKS
 					# But we might have omitted the encryption password question to skip encryption.
 					# In which case partition.encrypted will be true, but passwd will be false.
-					if partition.encrypted and (passwd := archinstall.arguments.get('!encryption-password', None)):
+					if partition.encrypted and (
+						passwd := archinstall.arguments.get('!encryption-password', None)
+					):
```

Comment:
Disclaimer (going to be a lot of these), this following is my personal opinion:

This decreases readability.